internal/webhook/namespace_deletion.go

```diff
@@ -0,0 +1,46 @@
+package webhook
```

Comment:
why is this webhook in internal , while the admission webhook is in api package? Seems a little bit strange: eventually they are both webhooks, each one doing their own safeguarding

Reply:
An upcoming PR will move the existing webhook code under `internal/webhook`. The `Module` webhook is under `api` because that is where `kubebuilder` generated it, but it is undesirable for many reasons - one of them is that people importing the `api` package now also import container-runtime.